src/core/Table/filters/FilterToggle.tsx

```diff
@@ -56,10 +56,11 @@ export const FilterToggle = <T extends Record<string, unknown>>(
             styleType='borderless'
             isActive={isVisible || column.filterValue}
             className={cx('iui-filter-button', className)}
-            onClick={(e: React.MouseEvent) => {
-              e.stopPropagation();
+            onClick={() => {
```

Comment:
have you checked if keyboard clicks work fine?

Reply:
Yes, it does work but we have bigger problems there like not being able to tab into the filter icon (it's hidden without hover						)

Reply:
it should be visible with focus though...?

Reply:
I was just to write the same. I think I never have tested keyboard navigation for opening filter menu or triggering sort...

Reply:
Created https://github.com/iTwin/iTwinUI-react/issues/488